javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,8 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  for (var i = 0; i < words.length; i++) {
+    var newWord = words[i] + suffix;
+    words[i] = newWord;
```

Comment:
Oh no!! You don't want to change the original array, unless it was the specific purpose of the function. Instead of changing the values in the original array, you should create a new array filled with the adjusted values.